site/docs/cow-and-mor.md

```diff
@@ -0,0 +1,195 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Copy-on-Write and Merge-on-Read
```

Comment:
Should this title just be `# Merge-on-Read`? Almost the entire file is covering merge-on-read and feels like it may only be mentioning copy-on-write in context of merge-on-read, in other words to help readers grasp the concept of merge-on-read. We could probably remove the `## Copy-on-write` sub-section entirely and place that one sentence that links to the spark-writes in the introduction.